bin/show_ecs_images

```diff
@@ -26,10 +18,39 @@ REGIONS=%w(
         ap-south-1
         sa-east-1
 )
+def show_image_id_for_amazon_linux(regions)
+  path = '/aws/service/ami-amazon-linux-latest/amzn-ami-hvm-x86_64-gp2'
+  STDERR.puts "getting image id of #{path}..."
 
-REGIONS.each do |region|
-  image_id=`aws --region #{region} ec2 describe-images --owners #{AMAZON_OWNER_ID} | jq -r '.Images[] | select(.ImageLocation|match("#{IMAGE_LOCATION}"))|.ImageId'`.chomp
-  puts %Q{        "#{region}"      => "#{image_id}",}
+  regions.each do |region|
+    image_id=`aws ssm get-parameters --names #{path} --region #{region} --query 'Parameters[0].Value' --output text`.chomp
+    puts %Q{        "#{region}"      => "#{image_id}",}
+  end
 end
 
 
+def show_image_id_for_ecs_optimized(regions)
+  path = '/aws/service/ecs/optimized-ami/amazon-linux/recommended'
```

Comment:
Same here, please make sure that this is **not** based on AL2

Reply:
I checked it by matching the output with [this page](https://docs.aws.amazon.com/ja_jp/AmazonECS/latest/developerguide/ecs-optimized_AMI.html).